pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -30,6 +30,19 @@ var (
 	Error = errs.Class("psdb")
 )
 
+// BwaStatus keep tracks of the BWA payout status
+type BwaStatus int32
+
+const (
+	// BwaStatusUNSENT sets the BWA payout status to UNSENT
+	BwaStatusUNSENT BwaStatus = 0
```

Comment:
Use `iota` for this - https://github.com/storj/storj/blob/953d77e9af5179b001e74dc6976ef9e4af7aeb8b/internal/sync2/fence.go#L32

Reply:
updated